source/elements/oneDAL/source/introduction.rst

```diff
@@ -1,3 +1,54 @@
 ============
 Introduction
 ============
+
+|dal_full_name| (|dal_short_name|) is a library that provides building blocks covering all stages of data analytics: data acquisition
```

Comment:
For some reasons, in the generated file I do not see 'oneAPI' Data Analytics Library. The rest looks good to me and can be resolved as the next steps

Reply:
fixed